draft-ecahc-moderation.md

```diff
@@ -191,21 +191,34 @@ IETF's various participation channels: a moderator team for the IETF.
 The creation of this team intends to address the issues identified
 in {{motiv}}.
 
-## Scope
+## Composition
+
+The moderator team will initially consist of no less than five
+individuals.  The IETF Chair appoints and replaces members of the
+moderator team.  In selecting members, the IETF Chair will take into
+account geographic coverage, expected and unexpected absenses, and
+team diversity.  The moderation team may expand or contract
+based on operational experience.  Apart from appointing and replacing
+moderators, the IETF Chair SHALL refrain from the day-to-day operation
```

Comment:
An insignificant nit, but I think that we should avoid RFC 2119 language since this isn't a protocol, and hence use the same words in lowercase.

Reply:
Let's be clear: there will be normative procedural statements here.

Reply:
We will take this up in another PR